ops/configs/nginx-prod.conf.template

```diff
@@ -0,0 +1,56 @@
+server {
+  listen 80;
+  return 301 https://${DOLLAR}host${DOLLAR}request_uri;
+}
+
+server {
+  listen 443 ssl;
+  server_name  $FRONTEND_HOST;
+  ssl_certificate /etc/nginx/certs/fullchain.pem;
+  ssl_certificate_key /etc/nginx/certs/privkey.pem;
+
+  location / {
+    proxy_pass   http://frontend:3000;
+  }
+}
+
+server {
+  server_name  $API_HOST;
+
+  listen 443 ssl;
+  ssl_certificate /etc/nginx/certs/fullchain.pem;
+  ssl_certificate_key /etc/nginx/certs/privkey.pem;
+
+  location / {
+    proxy_pass http://server:8000;
+
+    # from https://enable-cors.org/server_nginx.html
+    if (${DOLLAR}request_method = 'OPTIONS') {
+        add_header 'Access-Control-Allow-Origin' '*';
+        add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
+        #
+        # Custom headers and headers various browsers *should* be OK with but aren't
+        #
+        add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
+        #
+        # Tell client that this pre-flight info is valid for 20 days
+        #
+        add_header 'Access-Control-Max-Age' 1728000;
+        add_header 'Content-Type' 'text/plain; charset=utf-8';
+        add_header 'Content-Length' 0;
+        return 204;
+     }
+     if (${DOLLAR}request_method = 'POST') {
+        add_header 'Access-Control-Allow-Origin' '*';
+        add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
+        add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
+        add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
+     }
+     if (${DOLLAR}request_method = 'GET') {
```

Comment:
I don't see the difference between this and the previous `if`. What am I missing?

Reply:
you're correct they're identical. Nginx doesn't support "or" operators though so we need both if statements to handle post and get

Reply:
done